src/components/Loader/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+// Component abtract the logic of loading content from the server before displaying something,
+// Example Usage:
+// <Loader fetch={()=>fetch('www.google.com', {param: 1})}>
+//    {(data, isLoading)=>
+//      ... content here, check that `isLoading === false` before using `data`
+//    }
+// </Loader>
+export default class Loader extends React.Component {
+  state = {
+    isLoading: true,
+    data: null
+  };
+
+  async componentDidMount() {
+    this.setState({ isLoading: true });
+    let data = await this.props.fetch();
```

Comment:
`const`?

Reply:
I realized I have to use `const` more often :)